ts/Accessibility/Components/MenuComponent.ts

```diff
@@ -55,7 +55,8 @@ const {
  * @private
  */
 function getExportMenuButtonElement(chart: Chart): (SVGElement|undefined) {
-    return chart.exportSVGElements && chart.exportSVGElements[0];
+    return chart.exporting?.exportSVGElements &&
+        chart.exporting?.exportSVGElements[0];
```

Comment:
Optional chaining can be used on array indices too:
```suggestion
    return chart.exporting?.exportSVGElements?.[0];
```

Reply:
Corrected.